apps/web/components/eventtype/EventTypeDescriptionSafeHTML.tsx

```diff
@@ -1,11 +1,13 @@
 export type EventTypeDescriptionSafeProps = {
-  eventType: { description: string | null };
+  eventType: { description: string | null; isDynamic?: boolean };
 };
 
 export const EventTypeDescriptionSafeHTML = ({ eventType }: EventTypeDescriptionSafeProps) => {
   const props: JSX.IntrinsicElements["div"] = { suppressHydrationWarning: true };
   // @ts-expect-error: @see packages/prisma/middleware/eventTypeDescriptionParseAndSanitize.ts
   if (eventType.description) props.dangerouslySetInnerHTML = { __html: eventType.descriptionAsSafeHTML };
+  // Dynamic link description is not set/pulled from the DB, hence we can allow it here as is
+  if (eventType.isDynamic) props.dangerouslySetInnerHTML = { __html: eventType.description || "" };
```

Comment:
Since we're not setting or pulling the description from DB, but the `defaultEvents.ts` file, I believe we can exclude the description of dynamic events from the innerHTML safety check, no? @zomars @emrysal

Reply:
I'm not 100% sold on this. We can just change add `eventType.descriptionAsSafeHTML` to the defaultEvent and obtain the same result

Reply:
Fair enough. Will take care of that next 🙏

Reply:
@zomars here we go #4351 . Thanks for the suggestion, I should have thought of that. 🙏